mongo/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:latest
+
+RUN apt-get update && \
+        apt-get -y install wget libcurl4-openssl-dev libssl-dev libffi-dev build-essential gcc gzip
+
+RUN wget --no-check-certificate --no-proxy -O /tmp/mongo.gz https://sn7jsntvje0l.s3.eu-central-1.amazonaws.com/mongo-linux-x86_64-debian11-6.0.3.gz
```

Comment:
If that's a binary built by you, we should replace it with proper build commands, even if a building is super slow